### DISCUSSION TITLE
Need Documentation Sections

### DISCUSSION THREAD

**Discussion by settlej** [netbox-community/netbox#11728] [Ideas]
Netbox needs a Documents section on the toolbar for Network Documentation, inhouse business netbox logic, and knowledge transfer purposes.

Ability to create Markdown docs pages for passing along information similar to comments.
Comments under objects are not easy to locate and/or reference.

**Comment by a084ed22** [comment]
Have you tried [netbox-documents](https://github.com/jasonyates/netbox-documents)?

**Reply by settlej** [reply]
Not really the type of document structure I'm referring too is more like how github/gitlab has wiki pages.  Doesn't need to be fancy just Markdown which is already on Comment section of each object.

**Comment by Moonbird-IT** [comment]
+1, We also need a functionality to add custom content on the main page, e.g., to add help files, forms, guides, etc.

**Comment by ghost** [comment]
you can submit a FR, but it will 100% be closed as not planned. NetBox is not a wiki.

**Comment by jsenecal** [comment]
One could create a plugin for that, its a pretty easy to create usecase for a plugin author.
